#!/usr/bin/env python
PACKAGE = "hrsi_velocity_costmaps"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("costmap_resolution", double_t, 0, "The resolution of the costmap.", .05, 0,    1)
gen.add("min_costs", int_t, 0, "The minum costs assigned to the free area.", 0, 0, 100)
gen.add("max_costs", int_t, 0, "The maximum costs assigned to the forbidden area.", 100, 1, 100)

exit(gen.generate(PACKAGE, PACKAGE, "VelocityCostmaps"))
